server/interfaces/interfaces.go

```diff
@@ -1667,6 +1668,10 @@ type HitTrackerFactory interface {
 	NewCASHitTracker(ctx context.Context) HitTracker
 }
 
+type HitTrackerService interface {
+	Track(ctx context.Context, req *hitpb.TrackRequest) (*hitpb.TrackResponse, error)
+}
```

Comment:
Can we remove this and replace it with the generated proto service?

Reply:
oh yeah, duh